go.mod

```diff
@@ -6,121 +6,177 @@ require (
 	github.com/avast/retry-go v3.0.0+incompatible
 	github.com/btcsuite/btcd v0.22.1
 	github.com/btcsuite/btcutil v1.0.3-0.20201208143702-a53e38424cce
-	github.com/cosmos/cosmos-sdk v0.45.7
-	github.com/cosmos/ibc-go/v4 v4.0.0
+	github.com/cosmos/cosmos-sdk v0.47.0
+	github.com/cosmos/ibc-go/v7 v7.0.0
 	github.com/ethereum/go-ethereum v1.11.6
 	github.com/gogo/protobuf v1.3.3
```

Comment:
use `cosmos/gogoproto` instead of `gogo/protobuf`

Reply:
use `cosmos/gogoproto`
https://github.com/datachainlab/ethereum-ibc-relay-chain/pull/2/commits/21b222ebd3ee95cd7cdd146b6460e32badcbdeb9